### ISSUE TITLE
Case-split on holes

### ISSUE THREAD

**Issue by imkiva** [aya-prover/intellij-aya#10]

**Comment by ice1000** [comment]
Did you mean case-splits on bind patterns?

**Comment by imkiva** [comment]
> Did you mean case-splits on bind patterns?

Yes, but called from inside a hole (like `C-c C-c` in agda mode)

**Comment by ice1000** [comment]
> > Did you mean case-splits on bind patterns?
> 
> Yes, but called inside a hole (like `C-c C-c` in Agda mode)

We don't need to enforce quick-fixes to be in holes. We can make them happen everywhere. I think Agda mode does this due to their poor IDE support

**Comment by imkiva** [comment]
> > > Did you mean case-splits on bind patterns?
> > 
> > 
> > Yes, but called inside a hole (like `C-c C-c` in Agda mode)
> 
> We don't need to enforce quick-fixes to be in holes. We can make them happen everywhere. I think Agda mode does this due to their poor IDE support

Ohhh yes!
